tests/test_vcompressor.py

```diff
@@ -464,16 +464,17 @@ def _generate_test_data(self, test_file):
         with tempfile.TemporaryDirectory() as tmpdir:
             cvf_path = os.path.join(tmpdir, "result.cvf")
             dup_json_path = os.path.join(tmpdir, "result.json")
+            origin_json_path=get_tests_data_file_path(test_file)
```

Comment:
Sure, just add spaces around the `=`.

Reply:
OK，should I commit another PR after modify?

Reply:
No you should just push to this branch, the PR will be automatically updated.

Reply:
Sure, it seems to be something wrong on my internet,I will push it later,Thanks!

Reply:
I committed it on the github online editor,can you review it?